plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakePreferredPartitioning.java

```diff
@@ -135,13 +136,15 @@ private Session withForcedPreferredPartitioning()
         return Session.builder(getQueryRunner().getDefaultSession())
                 .setSystemProperty(USE_PREFERRED_WRITE_PARTITIONING, "true")
                 .setSystemProperty(PREFERRED_WRITE_PARTITIONING_MIN_NUMBER_OF_PARTITIONS, "1")
+                .setSystemProperty(TASK_PARTITIONED_WRITER_COUNT, "1")
```

Comment:
add a comment why this change is needed? Could we keep the default?

Reply:
> Could we keep the default?

Not really since we are testing the writer limit for partitions (100). So with a default of 32, we will never hit that limit.